lib/gruff/base.rb

```diff
@@ -103,6 +103,10 @@ class Base
     # Prevent drawing of line markers. Default is +false+.
     attr_writer :hide_line_markers
 
+    # Prevent drawing of column labels below a bar graph. For other graph types, hide_labels
+    # is ignored and continues to be tied to hide_line_markers. Default is +false+.
+    attr_writer :hide_labels
```

Comment:
If you want an accessor that is only effective for bar graphs, prepare it in `bar.rb`, not `base.rb`. 
I don't want to provide anything in `base.rb` that isn't commonly used.

Reply:
That makes sense.  I'll move the changes to bar.

Reply:
@Watson1978 DONE

Reply:
Thanks